ostd/src/arch/x86/boot/smp.rs

```diff
@@ -186,17 +191,27 @@ unsafe fn fill_boot_info_ptr(info_ptr: *const PerApRawInfo) {
 
 /// # Safety
 ///
-/// The caller must ensure the pointer to be filled is valid to write.
+/// 1. This function should only be called once on the BSP.
```

Comment:
This is a local helper method. It is not even an initialization method. I don't really understand why it can only be called once.